tests/test_surface_events.cpp

```diff
@@ -530,11 +532,21 @@ TEST_F(ClientSurfaceEventsTest, frame_timestamp_increases)
     surface.add_frame_callback(check_time_and_increment_count);
     wl_surface_commit(surface);
 
+    /* We don't need to wait for the server, but we *do* need
+     * the server to see this commit
+     */
+    client.flush();
+
     /**
-     * We need to sleep for multiple miliseconds to make sure the timestamp
-     * really does go up
+     * When run against a *real* compositor we should not need any
+     * delay here - when running on a real display, we would expect
+     * the second commit to wait for the next refresh cycle.
+     *
+     * But we're probably not running on a real display, so make
+     * things easier for the integration by waiting a simulated
+     * refresh cycle (at 60Hz) before submitting the next buffer.
      */
-    usleep(10000);
+    std::this_thread::sleep_for(std::chrono::milliseconds{17});
```

Comment:
**Suggestion**
Do we typically move magic numbers to constants? I am fine either way

Reply:
How about this?